repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/paths/ComputeModules.java

```diff
@@ -0,0 +1,66 @@
+package fr.inria.spirals.repairnator.process.step.paths;
+
+import fr.inria.spirals.repairnator.Utils;
+import fr.inria.spirals.repairnator.process.inspectors.ProjectInspector;
+import fr.inria.spirals.repairnator.process.inspectors.StepStatus;
+import fr.inria.spirals.repairnator.process.maven.MavenHelper;
+import fr.inria.spirals.repairnator.process.step.AbstractStep;
+import fr.inria.spirals.repairnator.states.PipelineState;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.building.ModelBuildingException;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class ComputeModules extends AbstractStep {
+
+    public ComputeModules(ProjectInspector inspector, boolean blockingStep) {
+        super(inspector, blockingStep);
+    }
+
+    private File[] findModules(String pomPath, boolean rootCall) {
+        List<File> modules = new ArrayList<>();
+
+        File pomFile = new File(pomPath);
+        try {
+            Model model = MavenHelper.readPomXml(pomFile, this.getInspector().getM2LocalPath());
```

Comment:
You have to prevent against NPE here: model can be null

Reply:
Fixed.